src/material/tree/testing/tree-harness.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {ComponentHarness, HarnessPredicate} from '@angular/cdk/testing';
+import {
+  MatNestedTreeNodeHarness,
+  MatTreeNodeHarness,
+  TreeHarnessFilters,
+  TreeNodeHarnessFilters
+} from '@angular/material/tree/testing';
+
+/** Harness for interacting with a standard mat-tree in tests. */
+export class MatTreeHarness extends ComponentHarness {
```

Comment:
This could maybe be a followup PR, but I think both the tree and the node harness should implement `HarnessLoader`